static/src/javascripts/projects/common/modules/support/articleCount.ts

```diff
@@ -0,0 +1,87 @@
+import { storage } from '@guardian/libs';
+import {
+	getWeeklyArticleHistory,
+	incrementWeeklyArticleCount,
+} from '@guardian/support-dotcom-components';
+import type { WeeklyArticleHistory } from '@guardian/support-dotcom-components/dist/shared/src/types/targeting';
+import { storageKeyDailyArticleCount } from 'common/modules/onward/history';
+import { getArticleCountConsent } from 'common/modules/support/supportMessaging';
+
+export interface DailyArticle {
+	day: number;
+	count: number;
+}
+
+type DailyArticleHistory = DailyArticle[];
```

Comment:
Personally I find these type aliases more confusing than just using `DailyArticle[]` which explains it's a list. I can't find a convention that we use though. 

Would `DailyArticleCount` be more descriptive?

Reply:
Fair point, I think I prefer `DailyArticle[]` as well! Though `WeeklyArticleHistory` (also an array) is defined in another repo so I'm inclined to keep it consistent here for now.
I have however just renamed `DailyArticle` -> `DailyArticleCount`